### ISSUE TITLE
CircleCI plugin broken

### ISSUE THREAD

**Issue by stefanalund** [spotify/backstage#1332]
The plugin does not render correctly:

![Screenshot 2020-06-17 at 10 23 12](https://user-images.githubusercontent.com/190856/84874248-a0502500-b084-11ea-9a2b-9f857584ddcb.png)

https://github.com/spotify/backstage/tree/master/plugins/circleci

**Comment by benjdlambert** [comment]
This is to do with the `react-router-v6` upgrade.

```
You rendered descendant <Routes> (or called `useRoutes`) at "/circleci" (under <Route path="/circleci">) but the parent route path has no trailing "*". This means if you navigate deeper, the parent won't match anymore and therefore the child routes will never render.

Please change the parent <Route path="/circleci"> to <Route path="/circleci/*">.
```

**Comment by shmidt-i** [comment]
Fixed by #1384
